neuronpp/core/cells/utils.py

```diff
@@ -18,4 +18,65 @@ def get_vecstim(ping_array):
     vec = h.Vector(ping_array)
     stim.play(vec)
 
-    return stim, vec
\ No newline at end of file
+    return stim, vec
+
+
+def get_spine_number(section, density, area_density):
```

Comment:
* I think that section param should be `Sec` indyferentny
* The method should have at least some docstring to express what it is doing and what params are and what are their units
* if area_density can be None or False -> express it as a default parameter eg. `area_density=None`

Reply:
I added docstrings and set area_density default value to False.